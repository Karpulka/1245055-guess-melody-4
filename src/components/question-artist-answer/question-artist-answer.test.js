import React from "react";
import renderer from "react-test-renderer";
import QuestionArtistAnswer from "./question-artist-answer.jsx";

const question = {
  type: `artist`,
  song: {
    answer: ``,
    src: ``
  },
  answers: [
    {
      answer: `one`,
      src: `pic-one`,
    },
    {
      answer: `two`,
      src: `pic-two`,
    },
    {
      answer: `three`,
      src: `pic-three`,
    },
  ]
};

const answer = {
  src: `pic-one`,
  answer: `one`
};

const handleChangeAnswer = () => {};

it(`Render QuestionArtistAnswer component`, () => {
  const tree = renderer
    .create(<QuestionArtistAnswer
      answer={answer}
      onChangeAnswer={handleChangeAnswer}
      question={question}
      id={0}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
