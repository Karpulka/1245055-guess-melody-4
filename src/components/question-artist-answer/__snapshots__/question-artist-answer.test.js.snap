// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render QuestionArtistAnswer component 1`] = `
<div
  className="artist"
>
  <input
    className="artist__input visually-hidden"
    id="answer"
    name="answer"
    onChange={[Function]}
    type="radio"
    value="one"
  />
  <label
    className="artist__name"
    htmlFor="answer"
  >
    <img
      alt="one"
      className="artist__picture"
      src="pic-one"
    />
    one
  </label>
</div>
`;
